pkg/controllers/subnet/subnet_controller.go

```diff
@@ -59,6 +61,19 @@ func (r *SubnetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 			}
 			log.V(1).Info("added finalizer on subnet CR", "subnet", req.NamespacedName)
 		}
+		vpcNetworkConfig := commonctl.ServiceMediator.GetVPCNetworkConfigByNamespace(obj.Namespace)
+		if vpcNetworkConfig == nil {
+			err := fmt.Errorf("failed to find VPCNetworkConfig for namespace %s", obj.Namespace)
+			log.Error(err, "operate failed, would retry exponentially", "subnet", req.NamespacedName)
```

Comment:
`operate failed`: cannot get configuration for Subnet CR

Reply:
We can also do this
log.Error(nil, "failed to find VPCNetworkConfig for Subnet CR", "subnet", req.NamespacedName, "namespace", obj.Namespace)

Reply:
Just saw it would return err in the end, so do we still need to log error here?

Reply:
OK